### PR TITLE
modifying code to train with two rafd type datasets

### PR DESCRIPTION
**Pull request by akanksha-atrey**

### CODE REVIEW

File: code/solver.py [hunk 1]
```diff
@@ -392,12 +369,12 @@ def train_multi(self):
...
-                    c_trg = label_trg.clone()
-                    zero = torch.zeros(x_real.size(0), self.c2_dim)
-                    mask = self.label2onehot(torch.zeros(x_real.size(0)), 2)
-                    c_org = torch.cat([c_org, zero, mask], dim=1)
-                    c_trg = torch.cat([c_trg, zero, mask], dim=1)
+                    c_org = self.label2onehot(label_org, self.c2_dim)
+                    c_trg = self.label2onehot(label_trg, self.c2_dim)
+                    zero = torch.zeros(x_real.size(0), self.c_dim)
+                    mask = self.label2onehot(torch.ones(x_real.size(0)), 2)
+                    c_org = torch.cat([zero, c_org, mask], dim=1)
```
Comment (vhissaria) [inline code/solver.py]: This should be torch.cat([c_org, zero, mask], dim=1), and even the next line

Comment (akanksha-atrey) [reply code/solver.py]: Oh you are right; I missed this. And self.c2_dim should be self.c_dim.
